src/openzaak/components/documenten/tasks.py

```diff
@@ -0,0 +1,377 @@
+
+    eio_serializer = EnkelvoudigInformatieObjectSerializer(
+        data=import_data, context={"request": request}
+    )
+
+    if not eio_serializer.is_valid():
+        error_message = (
+            "A validation error occurred while deserializing a "
+            f"EnkelvoudigInformatieObject on line {row_index}: \n"
+            f"{eio_serializer.errors}"
+        )
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    data: dict = eio_serializer.validated_data
+
+    if document_row.uuid:
+        try:
+            uuid = UUID(document_row.uuid, version=4)
+        except ValueError:
+            error_message = (
+                f"Given UUID for row {row_index} is not a valid UUID (version 4)"
+            )
+
+            logger.warning(error_message)
+            document_row.comment = error_message
+            document_row.processed = True
+
+            return document_row
+
+        if document_row.uuid in existing_uuids:
+            error_message = (
+                f"UUID given on row {row_index} was already found! Not overwriting "
+                "existing EIO."
+            )
+
+            logger.warning(error_message)
+            document_row.comment = error_message
+            document_row.processed = True
+
+            return document_row
+
+        data["uuid"] = str(uuid)
+    else:
+        data["uuid"] = str(uuid4())
+
+    gegevensgroep_fields = ("ondertekening", "integriteit")
+
+    for field in gegevensgroep_fields:
+        gegevens_groep_value = data.pop(field, {})
+
+        if not gegevens_groep_value:
+            continue
+
+        for key, value in gegevens_groep_value.items():
+            data[f"{field}_{key}"] = value
+
+    instance = EnkelvoudigInformatieObject(**data)
+    instance.canonical = EnkelvoudigInformatieObjectCanonical()
+
+    if not instance.identificatie:
+        instance.identificatie = generate_unique_identification(
+            instance, "creatiedatum"
+        )
+
+    zaak_id = document_row.zaak_id
+
+    if zaak_id and zaak_id not in zaak_uuids:
+        error_message = f"Zaak ID specified for row {row_index} is unknown."
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    try:
+        instance.clean()
+    except ValidationError as e:
+        error_message = (
+            "A validation error occurred while validating a "
+            f"EnkelvoudigInformatieObject on line {row_index}: \n"
+            f"{str(e)}"
+        )
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    file_path = document_row.bestandspad
+    path = Path(settings.IMPORT_DOCUMENTEN_BASE_DIR) / Path(file_path)
+
+    if not path.exists() or not path.is_file():
```

Comment:
I suspect we will have a lot of error with the path access before it's configured correctly. So let's check the existence of this folder in the very beginning of the importing process. 

And perhaps let's stop the process if the path doesn't exist, because there is a high change that all documents will have the same error?